pagopa/io-app
# feat(IT Wallet): [SIW-1853] Ipatente routing with IT Wallet

## Short description
This PR enables a link between the iPatente service and IT Wallet adding a custom route parsing the credentialType used by iPatente service.

## List of changes proposed in this pull request
- A new error screen was added when the user tries to navigate to the credential without having that specific credential.
- Added the navigation routing when using a specific deep link related to the iPatente service.

## How to test
Use this deep link to navigate into the credential (MDL only) and check these results:
- Wallet not active -> Error screen that route you to activate the wallet
- Wallet active but MDL not requested -> Screen that route you to request the MDL
- Wallet active MDL active -> MDL details screen

Deep link to be used: `ioit://itw/presentation/credential-detail/MDL`

Screenshot:
![IMAGE 2024-12-11 15:08:36](https://github.com/user-attachments/assets/ac182db6-273c-4163-bec7-670be8bb2b4d)
